netket/operator/_abstract_operator.py

```diff
@@ -140,7 +140,7 @@ def to_dense(self):
         Returns:
             numpy.matrix: The dense matrix representation of the operator.
         """
-        return self.to_sparse().todense()
+        return self.to_sparse().todense().A
```

Comment:
what is this .A? I can't find it in the numbly docs

Reply:
It's here: https://numpy.org/doc/stable/reference/generated/numpy.matrix.A.html#numpy.matrix.A, equivalent to `numpy.asarray(self)`.